src/modules/dom.js

```diff
@@ -134,11 +134,17 @@ export function getScrollTo(element: HTMLElement, offset: number): number {
  * @returns {HTMLElement|undefined}
  */
 export function getElement(element: string | HTMLElement): ?HTMLElement {
-  if (typeof element !== 'string') {
+  if (typeof element !== 'string' || !element) {
```

Comment:
If `!element` returns `element`? 😣

Reply:
It's true the original code returns `null` if element is `undefined`.